argus/recipees/cloud/windows.py

```diff
@@ -64,19 +64,34 @@ class WindowsCloudbaseinitRecipee(base.BaseCloudbaseinitRecipee):
     """Recipee for preparing a Windows instance."""
 
     def get_program_files(self):
-        """Get the location of program files from the instance."""
+        """Get the location of program files from the instance.
+
+        Warning: it's recommended to call this after cb-init is installed.
```

Comment:
Since we do use this method only to retrieve the location for the cloudbaseinit, why not change its name instead to get_cloudbaseinit_folder/location? Then you could drop the warning, which makes no sense in the context of get_program_files.